src/libraries/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj

```diff
@@ -248,6 +248,7 @@
   <ItemGroup>
     <ProjectReference Include="$(LibrariesProjectRoot)System.Text.RegularExpressions\gen\System.Text.RegularExpressions.Generator.csproj"
                       ReferenceOutputAssembly="false"
+                      SetTargetFramework="TargetFramework=netstandard2.0"
```

Comment:
I thought we would always reference the generators from a central targets file in our infra as opposed to having individual projects reference them explicitly, I'm curious why did we change that?

My main concern is that if new projects start adding a reference to the generator, they might forget to set this new property to pick the right TFM of the generator. If we instead have it be centralized, then it's less likely someone will trip on that in the future.

Reply:
I'm following the solution @ViktorHofer recommended.  I'm happy to change it if you both agree on a better way :)

Reply:
Responded below.